### PR TITLE
crm leadid api and select account from multi account

### PR DESCRIPTION
**Pull request by 065vaibhav**
1. create api for crm leadid
2. select single account from multiple account

### CODE REVIEW

File: liabilities/nre-Nro/nre-nroFunctions.js [hunk 1]
```diff
@@ -747,6 +761,291 @@ setTimeout(async (globals) => {
...
+      misCodeDetails: '',
+      identifierValue: `${day}/${month}/${year}`, // doubt in format
+      DoB: `${day}/${month}/${year}`, // multiple dob value
+      dateofBirth: `${day}/${month}/${year}`,
+      custBirthDate: `${day}/${month}/${year}`,
+      identifierName: '',
+      preferredChannel: '',
+      territoryName: '',
+      address: `${response?.txtCustadrAdd1} ${response?.txtCustadrAdd2} ${response?.txtCustadrAdd3}`,
+      companyName: 'ADOBE SYSTEMS INDIA PVT LTD',
```
Comment (roym1234) [inline liabilities/nre-Nro/nre-nroFunctions.js:785]: Remove hardcoded values

File: liabilities/nre-Nro/nre-nroFunctions.js [hunk 2]
```diff
@@ -747,6 +761,291 @@ setTimeout(async (globals) => {
...
+      presentAddressLine1: response.txtCustadrAdd1,
+      isIndianTaxResident: '1',
+      isTaxAddressSame: 'Yes',
+      isMailIDAvailable: 'Yes',
+      isPresentAddressSame: 'Yes',
+      isCommunicationAddressSame: 'Yes',
+      isPermanentAddressSame: 'Yes',
+      isSameAddress: 'Yes',
+      fatherNAme: response.customerFATCADtlsDTO[0].namCustFather,
+      employeeCategory: '2',
```
Comment (roym1234) [inline liabilities/nre-Nro/nre-nroFunctions.js:833]: Remove hardcoded Value

File: liabilities/nre-Nro/nre-nroFunctions.js (hunk 1 above)
Comment (065vaibhav) [reply liabilities/nre-Nro/nre-nroFunctions.js]: fixed

File: liabilities/nre-Nro/nre-nroFunctions.js (hunk 2 above)
Comment (065vaibhav) [reply liabilities/nre-Nro/nre-nroFunctions.js]: fixed and check

**Review by roym1234 — APPROVED** [review]

**Pull request merged**